server/punishments.ts

```diff
@@ -1063,7 +1079,7 @@ export const Punishments = new class {
 			if (!punishment) punishment = ['LOCK', user.locked, 0, ''];
 		}
 
-		const ticket = Chat.pages.help
+		const ticket = Chat.pages?.help
```

Comment:
?

Reply:
https://github.com/smogon/pokemon-showdown/pull/6338#discussion_r375246859

Reply:
Why can `Chat.pages` be null now?

Reply:
Oh, apparently it can. That sucks. :/